app/src/main/java/com/android/solvit/provider/ui/NotificationsScreen.kt

```diff
@@ -65,24 +121,96 @@ fun NotificationItem(
               .testTag("notificationItem_${notification.uid}"),
       colors =
           CardDefaults.cardColors(
-              containerColor = if (notification.isRead) Color.LightGray else Color.White),
+              containerColor =
+                  if (notification.isRead) colorScheme.surfaceVariant else colorScheme.background),
       elevation = CardDefaults.cardElevation(4.dp),
   ) {
     Column {
       Text(
           text = notification.title,
           modifier = Modifier.testTag("notificationTitle_${notification.uid}"),
           style = MaterialTheme.typography.titleMedium,
-          color = MaterialTheme.colorScheme.primary)
+          color = colorScheme.primary,
+          textAlign = TextAlign.Center)
       Text(
           text = notification.message,
           modifier = Modifier.testTag("notificationMessage_${notification.uid}"),
           style = MaterialTheme.typography.bodyMedium)
       Text(
-          text = "Received at: ${notification.timestamp.toDate()}",
+          text = "Received on: ${notification.timestamp.toDate().formatAsDate()}",
           modifier = Modifier.testTag("notificationTimestamp_${notification.uid}"),
           style = MaterialTheme.typography.titleSmall,
           color = Color.Gray)
     }
   }
 }
+
+fun Date.formatAsDate(): String {
+  val sdf = SimpleDateFormat("EEE, MMM d, yyyy 'at' h:mm a", Locale.getDefault())
+  return sdf.format(this)
+}
+
+@Composable
+fun Dialog(onDismiss: () -> Unit, serviceRequest: ServiceRequest) {
+  Dialog(onDismissRequest = onDismiss) {
```

Comment:
Consider using a less generic name for the Dialog

Reply:
Thanks for the feedback!